Incorrect application of the default rule

12:18     error  aws-default   default is not supported by API Gateway   servers[0].variables.basePath.default

This should only apply under $.components.schemas

--- a/aws_important_notes.yml
+++ b/aws_important_notes.yml
@@ -131,6 +131,7 @@ rules:
   aws-default:
     description: 'default is not supported by API Gateway'
     severity: error
-    given: $..default
+    given: $.components.schemas.*.properties.*
     then:
+      field: default
       function: undefined

